### PR TITLE
add: `wgcf`

### PR DESCRIPTION
**Pull request by wildyrando**
Adding wgcf to pacstall

### CODE REVIEW

File: packages/wgcf/wgcf.pacscript [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name="wgcf"
+pkgver="2.2.20"
+url="https://github.com/ViRb3/wgcf/releases/download/v${pkgver}/wgcf_${pkgver}_linux_amd64"
+hash="2a3ab38ace4d2284a3c810a28afcc0c5280d915372214246fe7da7245e06b0fa"
+arch=('amd64')
+pkgdesc="🚤 Cross-platform, unofficial CLI for Cloudflare Warp"
```
Comment (Elsie19) [inline packages/wgcf/wgcf.pacscript:6]: no emojis.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Comment by wildyrando** [comment]
Done, changed

File: packages/wgcf/wgcf.pacscript [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+hash="2a3ab38ace4d2284a3c810a28afcc0c5280d915372214246fe7da7245e06b0fa"
+arch=('amd64')
+pkgdesc="Cross-platform, unofficial CLI for Cloudflare Warp"
+maintainer="Wildy Sheverando <hai@wildyrando.com>"
+
+package() {
+  sudo mv wgcf_"${pkgver}"_linux_amd64 wgcf
+  sudo chmod +x wgcf
+  sudo mkdir -p "${pkgdir}/wgcf/usr/bin"
+  sudo mv wgcf "${pkgdir}/wgcf/usr/bin/"
```
Comment (Elsie19) [inline packages/wgcf/wgcf.pacscript:13]: You can make all these commands into one using `install`.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Comment by wildyrando** [comment]
Done, changed to using `install`

File: packages/wgcf/wgcf.pacscript [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name="wgcf"
+pkgver="2.2.20"
+url="https://github.com/ViRb3/wgcf/releases/download/v${pkgver}/wgcf_${pkgver}_linux_amd64"
+hash="2a3ab38ace4d2284a3c810a28afcc0c5280d915372214246fe7da7245e06b0fa"
+arch=('amd64')
+pkgdesc="Cross-platform, unofficial CLI for Cloudflare Warp"
+maintainer="Wildy Sheverando <hai@wildyrando.com>"
+
+package() {
+  sudo mkdir -p "${pkgdir}/usr/bin/"
```
Comment (Elsie19) [inline packages/wgcf/wgcf.pacscript:10]: you don't need this.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Comment by wildyrando** [comment]
Done, changed

**Review by Elsie19 — CHANGES_REQUESTED** [review]
This should be `wgcf-bin`.

**Comment by Elsie19** [comment]
change the package file name and path as well

**Comment by wildyrando** [comment]
> This should be `wgcf-bin`.

Solved

**Pull request closed without merging**